src/components/overview/overlay/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Modal } from '@mui/material';
+import ReactDOM from 'react-dom';
+
+interface OverlayProps {
+  children: React.ReactElement;
+  toggleModal: () => void;
+}
+
+export default function Overlay({ children, toggleModal }: OverlayProps) {
```

Comment:
Please read the doc how to use MUI modal window. Or look at the live example https://codesandbox.io/s/qfs62f?file=/Demo.tsx . if you use MUI you don't need to use Portals

Reply:
fix: Portals removed